src/js/view/view.js

```diff
@@ -698,6 +698,17 @@ define([
 
             // pass width, height from jwResize if present
             _resizeMedia(width, height);
+
+            if (_this.model.get('stretching') === 'uniform' && _preview) {
```

Comment:
...`&& _preview.height && _preview.width` otherwise `imageAspectRatio` will be `NaN`.